client/gutenberg/editor/edit-post/components/meta-boxes/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import MetaBoxesArea from './meta-boxes-area';
+
+function MetaBoxes( { location, isActive } ) {
```

Comment:
I'm wondering if we want to support meta boxes in Calypso at all :)

Reply:
Most likely not, I'm leaving the detailed clean up of the `edit-post` folder for the follow-up. :)